### PR TITLE
Progress bar and SIMD size

### PR DESCRIPTION
**Pull request by AndyAyersMS**
Set the SIMD size if the jit looks like it is cross-targeting. Currently
just checking altjits for "non".

Add simple progress bar to jit-diffs that shows up if you are running with
output to the console.

### CODE REVIEW

**Comment by AndyAyersMS** [comment]
@BruceForstall PTAL
cc @dotnet/jit-contrib 

jitdiffprogressbar

File: src/jit-dasm-pmi/jit-dasm-pmi.cs [hunk 1]
```diff
@@ -483,6 +483,14 @@ void GenerateAsmInternal()
                         generateCmd.EnvironmentVariable("COMPlus_AltJit", "*");
                         generateCmd.EnvironmentVariable("COMPlus_AltJitName", _altjit);
 
+                        // If this looks like a cross-targeting altjit, fix the SIMD size.
+                        // Here's one place where rationalized jit naming would be nice.
+                        if (_altjit.IndexOf("nonjit") > 0)
```
Comment (BruceForstall) [inline src/jit-dasm-pmi/jit-dasm-pmi.cs]: I guess this works... seems fragile...

File: src/jit-diff/diff.cs [hunk 2]
```diff
@@ -27,6 +20,58 @@ public enum DasmWorkKind
         Diff
     }
 
+    public class ProgressBar
```
Comment (BruceForstall) [inline src/jit-diff/diff.cs]: Fancy :-)

IMO, this might be "too much" progress output. seems like maybe once per second would be fine.

Comment (AndyAyersMS) [reply src/jit-diff/diff.cs]: I tried it at 5s and 1s initially and liked 0.1s better... Especially early, on when it seems not much is happening as we do some of the larger assemblies first.

Comment (briansull) [reply src/jit-diff/diff.cs]: Maybe it should be progressive, 0.1 for the first 2 seconds, then 1s seconds after that?

Comment (AndyAyersMS) [reply src/jit-diff/diff.cs]: Give this a try, and if it's annoying, we can work on improving it.

**Comment by AndyAyersMS** [comment]
Grr, my changes are not playing nice with one another...
```
- Finished 0/1 Base 0/1 Diff [0.1 sec]Setting SIMD Length to 16
/ Finished 1/1 Base 0/1 Diff [0.5 sec]
- Finished 1/1 Base 0/1 Diff [0.6 sec]Setting SIMD Length to 16
- Finished 1/1 Base 1/1 Diff [1.0 sec]
```
I suppose the "setting SIMD bit" could be something we put under verbose or in some RSP file.

**Pull request merged**